go/bundle/cmd/dump-bundle/main.go

```diff
@@ -23,6 +26,61 @@ func ReadBundleFromFile(path string) (*bundle.Bundle, error) {
 	return bundle.Read(fi)
 }
 
+func DumpExchange(e *bundle.Exchange, b *bundle.Bundle, verifier *signature.Verifier) error {
```

Comment:
Is it OK not to take io.Writer?

Reply:
I think stdout-only is fine for now, as user can redirect the output.